### PR TITLE
Test perf8

### PR DESCRIPTION
**Pull request by Paolob69**
Modified perf tests from Turbo-geth to erigon; add to README-md configuration parameters to be changed

### CODE REVIEW

File: tests/perf/README.md [hunk 1]
```diff
@@ -20,9 +28,9 @@ These are the instructions to execute *automatically* the performance comparison
 The command lines to activate such TG Core for performance testing are
 
 #### _TG Core_
```
Comment (canepat) [inline tests/perf/README.md:30]: TG -> Erigon

File: tests/perf/README.md [hunk 2]
```diff
@@ -20,9 +28,9 @@ These are the instructions to execute *automatically* the performance comparison
...
-From Turbo-Geth project directory:
+From Erigon project directory:
 ```
-build/bin/tg --goerli --private.api.addr=localhost:9090
+build/bin/erigon --goerli --private.api.addr=localhost:9090
 ```
 #### _TG RPCDaemon_
```
Comment (canepat) [inline tests/perf/README.md:35]: TG -> Erigon

File: tests/perf/README.md [hunk 3]
```diff
@@ -52,22 +60,27 @@ Vegeta request files are written to `/tmp/turbo_geth_stress_test`:
 From Silkrpc project directory check the performance test runner usage:
 ```
 $ tests/perf/run_perf_tests.py
-Usage: ./run_perf_tests.py vegetaPatternTarFile [daemonOnCore] [turboGethAddress] [turboGethHomeDir] [testRepetitions] [testSequence]
+Usage: ./run_perf_tests.py -h -p vegetaPatternTarFile -c daemonOnCore  -t turboErigonAddress -g turboErigonHomeDir -s silkrpcBuildDir -r testRepetitions - t testSequence
```
Comment (canepat) [inline tests/perf/README.md:63]: `turboErigonAddress` -> `erigonAddress`
`turboErigonHomeDir` -> `erigonHomeDir`

File: tests/perf/README.md [hunk 4]
```diff
@@ -52,22 +60,27 @@ Vegeta request files are written to `/tmp/turbo_geth_stress_test`:
...
-turboGethHomeDir         path to TG home folder (e.g. ../../../turbo-geth/)                               [default: ../../../turbo-geth/]
-testRepetitions          number of repetitions for each element in test sequence (e.g. 10)                [default: 10]
-testSequence             list of query-per-sec and duration tests as <qps1>:<t1>,... (e.g. 200:30,400:10) [default: 50:30,200:30,200:60,400:30]
+-h                      print this help
+-d rpcDaemonAddress     address of daemon eg (10.1.1.20)                                                       [default: localhost]
+-p vegetaPatternTarFile path to the request file for Vegeta attack                                             [default: ./vegeta/erigon_stress_test_001.tar]
+-c daemonVegetaOnCore   cpu list in taskset format for daemon & vegeta (e.g. 0-1:2-3 or 0-2:3-4 or 0,2:3,4...) [default: -:-]
+-a erigonAddress        address of ERIGON Core component as <address>:<port> (e.g. localhost:9090)             [default: localhost:9090]
+-g erigonHomeDir        path to ERIGON home folder (e.g. ../../../erigon/)                                     [default: ../../../erigon/build/]
```
Comment (canepat) [inline tests/perf/README.md:72]: The parameter name is `erigonHomeDir` that sounds like the Erigon project home folder as indicated in the example `(e.g. ../../../erigon/)` but then the default value is `[default: ../../../erigon/build/]` which seems like the build output directory. So either the parameter name is misleading or the default value is wrong here

File: tests/perf/README.md [hunk 5]
```diff
@@ -78,13 +91,13 @@ These are the instructions to execute *manually* the performance comparison test
 
 The command lines to activate such components for performance testing are listed below (you can also experiment allocating a different number of cores or removing `taskset`).
 
-#### _TG Core_
+#### _Erigon Core_
 From Turbo-Geth project directory:
```
Comment (canepat) [inline tests/perf/README.md:95]: Turbo-Geth -> Erigon

File: tests/perf/run_perf_tests.py [hunk 6]
```diff
@@ -29,16 +29,16 @@
 def usage(argv):
     """ Print script usage
     """
-    print("Usage: " + argv[0] + " -h -p vegetaPatternTarFile -c daemonOnCore  -t turboGethAddress -g turboGethHomeDir -s silkrpcBuildDir -r testRepetitions - t testSequence")
+    print("Usage: " + argv[0] + " -h -p vegetaPatternTarFile -c daemonOnCore  -t turboErigonAddress -g turboErigonHomeDir -s silkrpcBuildDir -r testRepetitions - t testSequence")
```
Comment (canepat) [inline tests/perf/run_perf_tests.py:32]: `turboErigonAddress` -> `erigonAddress`
`turboErigonHomeDir` -> `erigonHomeDir`

File: tests/perf/run_perf_tests.py [hunk 7]
```diff
@@ -29,16 +29,16 @@
...
     print("-d rpcDaemonAddress     address of daemon eg (10.1.1.20)                                                       [default: " + DEFAULT_RPCDAEMON_ADDRESS +"]")
     print("-p vegetaPatternTarFile path to the request file for Vegeta attack                                             [default: " + DEFAULT_VEGETA_PATTERN_TAR_FILE +"]")
     print("-c daemonVegetaOnCore   cpu list in taskset format for daemon & vegeta (e.g. 0-1:2-3 or 0-2:3-4 or 0,2:3,4...) [default: " + DEFAULT_DAEMON_VEGETA_ON_CORE +"]")
-    print("-a turboGethAddress     address of TG Core component as <address>:<port> (e.g. localhost:9090)                 [default: " + DEFAULT_TG_ADDRESS + "]")
-    print("-g turboGethHomeDir     path to TG home folder (e.g. ../../../turbo-geth/)                                     [default: " + DEFAULT_GETH_BUILD_DIR + "]")
+    print("-a erigonAddress        address of ERIGON Core component as <address>:<port> (e.g. localhost:9090)             [default: " + DEFAULT_ERIGON_ADDRESS + "]")
+    print("-g erigonHomeDir        path to ERIGON home folder (e.g. ../../../erigon/)                                     [default: " + DEFAULT_ERIGON_BUILD_DIR + "]")
```
Comment (canepat) [inline tests/perf/run_perf_tests.py:41]: Same comment as above for `erigonHomeDir`

**Review by canepat — CHANGES_REQUESTED** [review]

File: tests/perf/README.md (hunk 1 above)
Comment (Paolob69) [reply tests/perf/README.md:30]: ok

File: tests/perf/README.md (hunk 2 above)
Comment (Paolob69) [reply tests/perf/README.md:35]: ok

File: tests/perf/README.md (hunk 3 above)
Comment (Paolob69) [reply tests/perf/README.md:63]: ok

File: tests/perf/README.md (hunk 2 above)
Comment (Paolob69) [reply tests/perf/README.md:35]: fixed

File: tests/perf/README.md (hunk 4 above)
Comment (Paolob69) [reply tests/perf/README.md:72]: fixed

File: tests/perf/run_perf_tests.py (hunk 7 above)
Comment (Paolob69) [reply tests/perf/run_perf_tests.py:41]: ok

File: tests/perf/run_perf_tests.py (hunk 6 above)
Comment (Paolob69) [reply tests/perf/run_perf_tests.py:32]: ok

**Pull request merged**